package.json

```diff
@@ -35,7 +35,7 @@
     "buster-test": "~0.5",
     "buster-format": "~0.5.6",
     "http-server": "*",
-    "jscs": "~1.5.9"
+    "jscs": "~1.6.0"
```

Comment:
The ability to turn on/off linter rules for specific blocks of text was added in jscs 1.6, hence the need to bump the version.